helm/alfresco-content-services/templates/deployment-repository.yaml

```diff
@@ -11,6 +11,12 @@ metadata:
     component: repository
 spec:
   replicas: {{ .Values.repository.replicaCount }}
+  strategy:
+    type: Recreate
+    rollingUpdate: null
```

Comment:
@skopf: I believe as per [#24198](https://github.com/kubernetes/kubernetes/issues/24198#issuecomment-212730719), this would be an example of how we can implement `Recreate` strategy type.

Reply:
Ah, I see.
So when the live system has ```strategy.type==RollingUpdate```, then we cannot simply set it to ```"Recreate"```. We additionally need to ```null``` the ```strategy.rollingUpdate`` structure.
But with this change, we now always use ```strategy.type==Recreate``` without the possibility to overwrite it.
How about:
```
  strategy:
    type: {{ .Values.repository.updateStrategyType | default "RollingUpdate" }}
    rollingUpdate:  {{ .Values.repository.updateStrategyRollingUpdate }}
```
Would this be possible? Can we bring a whole dictionary from the values? Or is it only possible for scalar values?